include/TrustWalletCore/TWHRP.h

```diff
@@ -38,6 +39,7 @@ static const char *_Nonnull HRP_DIGIBYTE = "dgb";
 static const char *_Nonnull HRP_GROESTLCOIN = "grs";
 static const char *_Nonnull HRP_LITECOIN = "ltc";
 static const char *_Nonnull HRP_QTUM = "qc";
+static const char *_Nonnull HRP_TERRA = "terra";
```

Comment:
Do we still need `TWHRP` enum when we have `hrp` in `coins.json`? It would be ideal that `hrp` is the single source of truth.

Reply:
This values are generated from `coins.json`

Reply:
Ah we just did it, makes sense, my bad.
Somehow I would think it won’t be tracked in Git, and be moved to generated?